corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/javascript/hqwebapp/ui_elements/ui-element-select.js.diff.txt

```diff
@@ -10,3 +10,14 @@
  ], function (
      $,
      _,
+@@ -80,8 +80,8 @@
+         setOptions: function (options) {
+             this.options = options.map(o => _.isString(o) ? {value: o, label: o} : o);
+             this.$edit_view.html('');
+-            for (let i = 0; i < this.options.length; i += 1) {
+-                const option = this.options[i];
++            for (var i = 0; i < this.options.length; i += 1) {
++                var option = this.options[i];
+                 if ('groupName' in option) {
+                     $(`<optgroup label="${option.groupName}"/>`).appendTo(this.$edit_view);
+                 } else {
```

Comment:
I think this diff indicates that you should make the same changes to the 'bootstrap3' version of the `ui-element-select.js` file.

Reply:
Yes, I made the changes to both bootstrap 3 and 5